resource_retriever/package.xml

```diff
@@ -12,8 +12,9 @@
 
   <author email="jfaust@willowgarage.com">Josh Faust</author>
   <author email="isucan@gmail.com">Ioan Sucan</author>
```

Comment:
I think the author tags are supposed to be below the license tag, but I'll approve anyways since it's a preexisting issue

https://github.com/ros-infrastructure/rep/blob/3486e2328c83e115191e7de4335e22c721daccf9/xsd/package_format3.xsd#L66